sdk/servicebus/azure-servicebus/azure/servicebus/aio/_servicebus_client_async.py

```diff
@@ -216,3 +217,53 @@ def get_queue_receiver(self, queue_name, **kwargs):
         )
 
         return receiver
+
+    def get_queue_session_receiver(self, queue_name, session_id=None, **kwargs):
```

Comment:
get_subscription_session_receiver?

Reply:
There was a slight game of telephone between your sub changes going in and this split occurring.  I think it's been rectified as of the latest commit. (which entertainingly I think we crossed-each-other-in-flight for my pushing that and your making these comments :P)